notes/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from uuid import uuid4
+# Create your models here.
+
+class Note(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid4,editable=False)
+    title = models.CharField(max_length=200)
+    content = models.TextField(blank=True)
+
```

Comment:
Model implementation looks good. A little info about inheritance, Class Note(models.Model) means Note is inheriting from models.Model.